virttest/virt_vm.py

```diff
@@ -659,6 +659,7 @@ def get_address(self, index=0):
             raise VMMACAddressMissingError(index)
         if self.ip_version == "ipv4":
             # Get the IP address from arp cache
+            del self.address_cache[nic.mac.lower()]
```

Comment:
@sathnaga Setting ip to None maybe cover the ip has updated in address_cache, so I think shoud modify it in this code path.